tests/Aspire.Hosting.MySql.Tests/MySqlFunctionalTests.cs

```diff
@@ -442,4 +442,53 @@ await pipeline.ExecuteAsync(async token =>
             Assert.Equal("BatMobile", cars[0].Brand);
         }, cts.Token);
     }
+
+    [Fact]
+    [RequiresDocker]
+    public async Task MySql_WithPersistentLifetime_ReusesContainers()
+    {
+        var cts = new CancellationTokenSource(TimeSpan.FromMinutes(10));
+
+        using var builder1 = TestDistributedApplicationBuilder.Create(testOutputHelper);
+        var resource1 = builder1.AddMySql("resource").WithLifetime(ContainerLifetime.Persistent);
```

Comment:
We should also add WithPhpMyAdmin. And simiarly WithPgWeb and WithPgAdmin for Postgres. Ensure all the containers are reused across runs.

Reply:
Done, also added the variants that trigger different config types.